### PR TITLE
Display listmap

### PR DESCRIPTION
**Pull request by annamarika**

### CODE REVIEW

File: src/components/Product/ProductGrid.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+const ProductUl = styled.ul`
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+`;
+
+const ProductLi = styled.li`
+	list-style: none;
+`;
```
Comment (UrsulaMontag) [inline src/components/Product/ProductGrid.js:24]: Maybe you want to reuse the list-style an another point? Think about creating a separate styled-component for the list?

File: src/components/Product/ProductGrid.spec.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+		const showText = screen.getByText(
+			/Cute red dress. Perfect for a summer party. Almost new. Size 36. Brand Zara. Cologne./i
+		);
+		const showButton = screen.getByRole('button');
+
+		expect(showText).toBeInTheDocument();
+		expect(showButton).toBeInTheDocument();
+	});
+});
```
Comment (UrsulaMontag) [inline src/components/Product/ProductGrid.spec.js:28]: Maybe you want to test if the number of displayed products is correct? Think of array.length.

**Review by UrsulaMontag — COMMENTED** [review]
Hey Anna,
I reviewed your nice product-list and I like it so much. I left a few comments you can think about. But there is the comment about the styled list, which I think you should revise in any case.
I would also like to point out that you are still at the beginning of your project. if you don't subdivide your components folder further in your file structure, it will certainly become very confusing over time. I suggest to divide your components folder into further folders, e.g. for the individual components and one for ui.

File: src/components/Product/ProductGrid.js (hunk 1 above)
Comment (annamarika) [reply src/components/Product/ProductGrid.js:24]: Thanks Ursula, my next  UserStory is about reorganize my styled component files. I started the project with a misunderstanding on how to work properly with styled comments. I already started this branch when I got the noticed of the misleading style component folder structure. So I would leave the List item to the next branch.

File: src/components/Product/ProductGrid.spec.js (hunk 2 above)
Comment (annamarika) [reply src/components/Product/ProductGrid.spec.js:28]: Thanks I will try this

**Review by UrsulaMontag — APPROVED** [review]

**Pull request merged**